app/change_set_persisters/plum_change_set_persister/delete_fixity_check.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+class PlumChangeSetPersister
+  class DeleteFixityCheck
```

Comment:
Maybe we should just store the fixity properties on the FileSet - maybe in a nested object to keep them nicely contained, if that matters? That way we don't have to worry about this - the fixity check will just be attached to the life of the fileset.

Reply:
if it's nested doesn't that means it doesn't get its own created / updated dates? we are going to need those.

Reply:
well updated, at least. I guess that could be managed manually.